content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from "shared/components/common/PointOfInterest";
+
+# Build Edge Artifacts using Content Bundle
+
+Palette's Edge solution supports creating Edge artifacts for edge devices having low or zero internet bandwidth or deployed in an *air-gapped* environment. An *air-gapped* environment is a deployment site with no direct or indirect connectivity to other devices or outside networks. In such environments, you prepare edge devices using the Edge artifacts built using a *content bundle*.
+
+
+A content bundle is an archive that includes the operating system image, Kubernetes distribution, additional layers, and packs and manifest files specified in your cluster profile that you want to deploy to the Edge cluster. A content bundle provides several benefits, such as:
+<br />
+
+- Preloads required software dependencies to remove the need for downloading assets during cluster deployment, 
+
+
+- Optimizes the deployment process in bandwidth-constrained environments 
+
+
+- Installs only authorized tools or software on Edge devices. 
+
+The current guide provides instructions to create and use a content bundle to build the Edge installer ISO image. However, before you proceed with the current guide, you must build provider images by following the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) guide. 
+<br />
+
+<WarningBox>
+
+This guide extends the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) workflow. Therefore, you must follow the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) guide to build the provider images for all the Palette-supported Kubernetes versions and push one of them to an image registry of your choice. This guide uses a provider image compatible with K3s v1.25.x pushed to the [ttl.sh](https://ttl.sh/) image registry. However, you can use any other provider image or registry per your requirements.
+
+</WarningBox>
+<br />
+
+In this guide, you will begin with installing the necessary tool, the Palette Edge CLI, on your development machine. Next, you will create a cluster profile in Palette referencing one of the provider images you built using the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) workflow. Then, you will access the cluster profile using the Palette Edge CLI utility to create a content bundle. Lastly, when your content bundle is ready, you will use the CanvOS utility to embed the content bundle, site-specific configuration, and user data into the Edge installer ISO image.  
+  
+
+The diagram below displays the high-level steps to build the Edge installer ISO image using a content bundle.
+
+![An overarching diagram displaying the workflow in the current guide.](/clusters_edge_edge-forge-workflow_build-images_build-artifacts_overarching.png)
+
+
+# Prerequisites
```

Comment:
Besides the resource prereqs (CPU, Mem, Storage). You can make the bullet below the first bullet and then simplify all the other bullets without mentioning the previous guide again. 


* To complete this guide, you must have completed the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) guide and generated all the required Edge artifacts. The required Edge artifacts are an Edge Installer ISO, and provider images created and uploaded to an image registry.